controllers/base_controller.py

```diff
@@ -46,6 +46,10 @@ def __init__(self, *args, **kw):
                 if self.response:
                     self.response.headers['Vary'] = 'Accept-Encoding'
 
+    @property
+    def logged_in(self):
+        return self._user_bundle.user is not None
+
```

Comment:
@fangeugene PTAL - does this seem okay? I want to know if a user is signed in in the `CacheableHandler` so we can conditionally hide/show the notification testing boxes on the webhooks page

Reply:
This doesn't work, since the cached version could think that a user is signed in.

The correct thing to do is to not use `CacheableHandler` and just use `LoggedInHandler`

Reply:
Thanks! Made this change and that seems to work.